.github/workflows/test.yml

```diff
@@ -57,8 +57,12 @@ jobs:
         run: dotnet build --no-restore -c Release
       - name: Build solution [Debug]
         run: dotnet build --no-restore -c Debug
-      - name: Test solution [Debug]
+      - name: Test solution [Debug] with code coverage
+        if: ${{ startsWith(matrix.os, 'ubuntu') }}
         run: dotnet test --no-restore --no-build -p:CollectCoverage=true
+      - name: Test solution [Debug] without code coverage
```

Comment:
Interesting. Some problems or just enchancement?

Reply:
https://github.com/graphql-dotnet/parser/pull/266

Reply:
No idea why but won’t pass signature checks with code coverage enabled. Since signatures are only verified on .NET, and as .NET only runs on Windows, and since we don’t upload coverage reports on Windows anyway, I just disabled code coverage calculations on Windows. 

I suspect that .NET or something must inject code into the loaded dlls during code coverage reports, which breaks the signature, but I am entirely just guessing. I can reproduce the issue locally and can verify that the dlls have valid signatures — in fact the exact same dlls that pass without code coverage enabled fail when code coverage is enabled.

Reply:
It does show that signatures are working properly however!

Reply:
To reproduce,
1. `dotnet build`
2. `dotnet test --no-restore --no-build` which passes
3. `dotnet test --no-restore --no-build -p:CollectCoverage=true` which fails due to signature failure

Reply:
It is Coverlet instrumentation.